core/src/test/scala/unit/kafka/server/DescribeQuorumIntegrationTest.scala

```diff
@@ -80,6 +85,36 @@ class DescribeQuorumRequestTest(cluster: ClusterInstance) {
     assertTrue(leaderState.logEndOffset > 0)
   }
 
+  @ClusterTest
+  def testDescribeQuorumRequestToBrokers() = {
```

Comment:
Looking at the other tests, this test suite does not seem to be the right place for this test. I wonder if `KRaftClusterTest` would be more appropriate for instance but I don't feel strong about this.